src/Parser/DemonHunter/Havoc/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,91 @@
+import SPELLS from 'common/SPELLS';
+
+import CoreChecklist, { Rule } from 'Parser/Core/Modules/Features/Checklist';
+import Abilities from 'Parser/Core/Modules/Abilities';
+import { PreparationRule } from 'Parser/Core/Modules/Features/Checklist/Rules';
+import { GenericCastEfficiencyRequirement } from 'Parser/Core/Modules/Features/Checklist/Requirements';
+import CastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+import LegendaryUpgradeChecker from 'Parser/Core/Modules/Items/LegendaryUpgradeChecker';
+import LegendaryCountChecker from 'Parser/Core/Modules/Items/LegendaryCountChecker';
+import PrePotion from 'Parser/Core/Modules/Items/PrePotion';
+import EnchantChecker from 'Parser/Core/Modules/Items/EnchantChecker';
+
+import AlwaysBeCasting from './AlwaysBeCasting';
+
+class Checklist extends CoreChecklist {
```

Comment:
This Checklist style is deprecated, could you please convert it to the new style?

Reply:
Sure! I'll do it asap.